praw/models/reddit/live.py

```diff
@@ -34,15 +34,28 @@ def __call__(self):
         return temp if isinstance(temp, RedditorList) else temp[0]
 
     def __init__(self, thread):
-        """Create a LiveContributorRelationship instance.
+        """Create a :class:`.LiveContributorRelationship` instance.
 
         :param thread: An instance of :class:`.LiveThread`.
 
+        .. note:: This class should not be initialized directly instead obtain
```

Comment:
Updated. Because this sentence is copied from subreddit.py, I added fix for that.